tests/cmds/test_alerts.py

```diff
@@ -789,6 +789,52 @@ def test_send_to_when_given_ignore_cert_validation_uses_certs_equal_to_ignore_st
     )
 
 
+@pytest.mark.parametrize("protocol", (ServerProtocol.UDP, ServerProtocol.TCP))
```

Comment:
These tests are good examples of when parameterized tests make sense.

Reply:
Right!